src/main/groovy/life/qbic/datamodel/dtos/business/Offer.groovy

```diff
@@ -91,6 +91,10 @@ class Offer {
      * The net price of all items for which an overhead cost is not applicable, without overhead and taxes
      */
     final double itemsWithoutOverheadNetPrice
+    /**
+     * The overhead modifier applied to the pricing dependent on the customer affiliation
+     */
+    final double overheadModifier
```

Comment:
Is this really offer specific? Isn't it dependent on the affiliation?

Reply:
The applicable overhead modifier is dependent on the affiliation stored in the Offer. 
I could also add a small method to the OfferPDF Conversion that extracts the affilation and then assigns the overhead cost. However we already have a method in the Offer Entity in the Offer-manager that determines the overhead modifier so i thought it would be better to avoid duplicate methods.

Reply:
I think it makes more sense to add the method that extracts that property since it does not belong to the offer but the affiliation associated with the customer in the offer.

Reply:
But then that means as soon as the overhead cost or affiliation change you would have to update it seperately in the Offer2PDF converter class and in the Offer entitity